ddoser/Dockerfile

```
@@ -0,0 +1,7 @@
+FROM python:3.7.3-stretch
+
+WORKDIR /usr/src/app
+COPY . .
+RUN pip install --trusted-host pypi.org --trusted-host files.pythonhosted.org --upgrade pip
+RUN pip install --trusted-host pypi.org --trusted-host files.pythonhosted.org -r requirements.txt
```

```suggestion
RUN pip install --trusted-host pypi.org --trusted-host files.pythonhosted.org --upgrade pip &&\
    pip install --trusted-host pypi.org --trusted-host files.pythonhosted.org -r requirements.txt
```

That'll have the same effect but will produce a smaller container image.